Modules/xSharePoint/DSCResources/MSFT_xSPVisioServiceApp/MSFT_xSPVisioServiceApp.schema.mof

```diff
@@ -0,0 +1,18 @@
+/*
+**Description**
+
+This resource is responsible for creating Visio Graphics Service Application instances within the local SharePoint farm.
+The resource will provision and configure the Visio Graphics Service Application.
+**Example**
+    xSPVisioServiceApp VisioServices
+    {
+        Name = "Visio Graphics Service Application"
+        ApplicationPool = "SharePoint Web Services"
+    }
+*/
+[ClassVersion("1.0.0.0"), FriendlyName("xSPVisioServiceApp")]
+class MSFT_xSPWebApplication : OMI_BaseResource
```

Comment:
Thanks I will double check. I actually believe I copied the bcsserviceapp schema to start, so problem will probably exist there as well, I'll double check

Nik Charlebois
Microsoft MVP
From: Brian Farnhill
Sent: jeudi 3 décembre 2015 3:24 AM
To: PowerShell/xSharePoint
Reply To: PowerShell/xSharePoint
Cc: Nik Charlebois
Subject: Re: [xSharePoint] Added xSPVisioServiceApp (#111)


> @@ -0,0 +1,18 @@
> +/*
> +**Description**
> +
> +This resource is responsible for creating Visio Graphics Service Application instances within the local SharePoint farm.
> +The resource will provision and configure the Visio Graphics Service Application.
> +**Example**
> +    xSPVisioServiceApp VisioServices
> +    {
> +        Name = "Visio Graphics Service Application"
> +        ApplicationPool = "SharePoint Web Services"
> +    }
> +*/
> +[ClassVersion("1.0.0.0"), FriendlyName("xSPVisioServiceApp")]
> +class MSFT_xSPWebApplication : OMI_BaseResource

The class name doesn't match - its copied from xSPWebApplication - should be MSFT_xSPVisioServiceApp

---
Reply to this email directly or view it on GitHub:
https://github.com/PowerShell/xSharePoint/pull/111/files#r46523792